doc/man7/crypto.pod

```diff
@@ -21,6 +21,57 @@ agreement, certificate handling, cryptographic hash functions, cryptographic
 pseudo-random number generators, message authentication codes (MACs), key
 derivation functions (KDFs), and various utilities.
 
+=head2 Symmetric encryption
```

Comment:
This is very good idea. I would just put it after the `Operations` section

Reply:
I also think this is a great idea - we desperately need some of this "overview" and "introductory" type content. But we do need to consider the ordering of where this text sits in the man page. I think we should really be defining some fundamental OpenSSL terminology first (i.e. algorithms, operations, providers etc). That terminology is probably essential to understand before going off to read any of the other links and pages that you are providing below. This content should be moved further down IMO